### PR TITLE
[FLINK-11613][chinese-translation,Documentation]Translate the "Project Template for Scala" page into Chinese

### PR DESCRIPTION
**Pull request by Tom-Goong**
## What is the purpose of the change
This pull request completes the Chinese translation of "Project Template for Scala" page from official document.

## Brief change log
* Translate the "Project Template for Scala" page into Chinese

## Verifying this change
This change is to add a new translated document.

## Does this pull request potentially affect one of the following parts:
* Dependencies (does it add or upgrade a dependency): no
* The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
* The serializers: no
* The runtime per-record code paths (performance sensitive): no
* Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
* The S3 file system connector: no

## Documentation
* Does this pull request introduce a new feature? no

### CODE REVIEW

**Comment by klion26** [comment]
@Tom-Goong thank you for your contribution, we have to wait until the base pr #7754 has been merged first.

**Comment by Tom-Goong** [comment]
> @Tom-Goong thank you for your contribution, we have to wait until the base pr #7754 has been merged first.

Ok, thanks for your comment.
I just started making some simple contributions to the open source project, and the process is still unfamiliar to me. If there is something wrong, let me know. Also, hope that will be more contributions I can do.

**Comment by Tom-Goong** [comment]
@klion26  Ok, thanks for your comment.
I just started making some simple contributions to the open source project, and the process is still unfamiliar to me. If there is something wrong, let me know. Also, hope that will be more contributions I can do.

**Comment by wuchong** [comment]
Hi @Tom-Goong , thanks for your contribution. 

Could you rebase your commit to the latest master? So that We can start to review it.

**Comment by Tom-Goong** [comment]
Hi @wuchong, sorry, i failed to rebase my commit to the first place. so I've checkout a new branch and update origin by force

**Comment by wuchong** [comment]
Thanks @Tom-Goong , I will review it today.

**Comment by wuchong** [comment]
@flinkbot approve-until quality

**Comment by wuchong** [comment]
@flinkbot disapprove quality

File: docs/dev/projectsetup/scala_api_quickstart.zh.md [hunk 1]
```diff
@@ -176,55 +172,50 @@ quickstart/
...
 
-If you are writing a streaming application and you are looking for inspiration what to write,
-take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program)
+如果你准备编写流处理应用，正在寻找灵感来写什么，
+可以看看[流处理应用指南]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program)
 
-If you are writing a batch processing application and you are looking for inspiration what to write,
-take a look at the [Batch Application Examples]({{ site.baseurl }}/dev/batch/examples.html)
+如果你准备编写批处理应用，正在寻找灵感来写什么，
+可以看看[批处理应用样例]({{ site.baseurl }}/dev/batch/examples.html)
```
[the remark this answers is not in the document]
Comment (Tom-Goong) [reply docs/dev/projectsetup/scala_api_quickstart.zh.md]: Great choice

File: docs/dev/projectsetup/scala_api_quickstart.zh.md [hunk 2]
```diff
@@ -176,55 +172,50 @@ quickstart/
...
-## Next Steps
+## 下一步
 
-Write your application!
+开始编写你的应用！
 
-If you are writing a streaming application and you are looking for inspiration what to write,
-take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program)
+如果你准备编写流处理应用，正在寻找灵感来写什么，
+可以看看[流处理应用指南]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program)
```
[the remark this answers is not in the document]
Comment (Tom-Goong) [reply docs/dev/projectsetup/scala_api_quickstart.zh.md]: Great choice

File: docs/dev/projectsetup/scala_api_quickstart.zh.md [hunk 3]
```diff
@@ -83,47 +83,45 @@ fork in run := true
...
 > eclipse
 {% endhighlight %}
 
-Now you can import the project into Eclipse via `File -> Import... -> Existing Projects into Workspace` and then select the project directory.
+现在你可以通过 `File -> Import... -> Existing Projects into Workspace` 将项目导入 Eclipse，然后选择项目目录。
 
 ## Maven
 
-### Requirements
+### 要求
```
[the remark this answers is not in the document]
Comment (Tom-Goong) [reply docs/dev/projectsetup/scala_api_quickstart.zh.md]: agree

**Comment by Tom-Goong** [comment]
Hi, @wuchong , thanks for great help, all comments have been updated.

**Comment by wuchong** [comment]
LGTM.

Merging...

**Pull request closed without merging**